internal/configs/ingress_test.go

```diff
@@ -568,6 +568,126 @@ func TestGenerateNginxCfgForMergeableIngressesForBasicAuth(t *testing.T) {
 	}
 }
 
+func TestGenerateNginxCfgForLimitReq(t *testing.T) {
+	t.Parallel()
+	cafeIngressEx := createCafeIngressEx()
+	cafeIngressEx.Ingress.Annotations["nginx.org/limit-req"] = "200r/s"
```

Comment:
What happens if the suffix 'r/s' or 'r/m' is omitted?
ref: [NGINX docs](https://nginx.org/en/docs/http/ngx_http_limit_req_module.html#limit_req)

```shell
The rate is specified in requests per second (r/s). If a rate of less than one request per second is desired, it is specified in request per minute (r/m). For example, half-request per second is 30r/m.
```

Is validation of the string representing number of requests required?

cc @haywoodsh @vepatel